sdk/file_sdk/UnitTests/zip_ut/zip_ut.pro

```diff
@@ -0,0 +1,30 @@
+REPO_ROOT = $$PWD/../../../..
+DESTDIR   = $$REPO_ROOT/bin/tests
+TARGET = zip_tests
+
+QT       += testlib
+
+QT       -= gui
+
+CONFIG   += console c++11 debug_and_release resources_big
```

Comment:
>resources_big [](start = 44, length = 13)

do you have a resource file?

Reply:
now yes, thanks for the help :) 

---
In reply to: [139356990](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139356990) [](ancestors = 139356990)